cognite/neat/rules/exceptions.py

```diff
@@ -2022,6 +2022,24 @@ def __init__(self, missing_fields: set[str], verbose: bool = False):
         super().__init__(self.message)
 
 
+class ClassNoProperties(NeatException):
```

Comment:
In principle you might have class without a property if that class has a parent. Example GeneratingUnit -> WindGeneratingUnit(GeneratingUnit), so basically either Class must have:
- either properties
- or parent(s)
- or both parent(s) and properties